addOns/quickstart/src/main/java/org/zaproxy/zap/extension/quickstart/AttackPanel.java

```diff
@@ -394,6 +395,13 @@ private void stopAttack() {
         stopButton.setEnabled(false);
     }
 
+    void setButtonStates(boolean val) {
```

Comment:
- e.g. `private void setSpiderButtonsEnabled(boolean enabled)`
- Since this is not including the other buttons the `complete` case can be changed to fall-through and avoid the duplication.